Dockerfile

```diff
@@ -56,17 +53,21 @@ ENV \
   LNDR_UCAC_EUR="0xae0d61120070411bf96b1d0d42fe9f4023e9f8f1" \
   LNDR_UCAC_GBP="0x5c433119ec13fdac42359838f7df93124d054d0c" \
   LNDR_UCAC_HKD="0xb52a2db8ae67a51b1906e4b333930641d62272fd" \
+  LNDR_UCAC_IDR="0xfc0f4fe61f23ca895f7e8a51d4e08462f4926687" \
+  LNDR_UCAC_ILS="0x77ee746c16a817109567b895ff9b9a75bf354bf4" \
+  LNDR_UCAC_JPY="0x9d9462f70067f506ac26bd523222f4f8020924d4" \
+  LNDR_UCAC_KRW="0x9945a5b005a898a435adf30fe88f2818ccc0ba5c" \
+  LNDR_UCAC_MYR="0x79a5f8d6cc432c1f017648c4fae840dff4cfcaf2" \
   LNDR_UCAC_NOK="0x943b8e14145692f33082ca062ee48be49d48d476" \
   LNDR_UCAC_NZD="0x28a2997d3c21087053b7bb5c244bf06bf84c984e" \
+  LNDR_UCAC_RUB="0x14eb816e20af23ef81cc1deeba71d8642edcb621" \
   LNDR_UCAC_SEK="0x55b1d21c802e74b5e2230e9f3af28d22f8128ddd" \
-  LNDR_UCAC_IDR="0xfc0f4fe61f23ca895f7e8a51d4e08462f4926687" \
-  LNDR_UCAC_MYR="0x79a5f8d6cc432c1f017648c4fae840dff4cfcaf2" \
   LNDR_UCAC_SGD="0x3ac772c0f927df3c07cd90c17b536fcab86e0a53" \
   LNDR_UCAC_THB="0x9a76e6a7a56b72d8750f00240363dc06d09c7161" \
-  LNDR_UCAC_VND="0x815dcbb2008757a469d0daf8c310fae2fc41e96b" \
-  LNDR_UCAC_ILS="0x77ee746c16a817109567b895ff9b9a75bf354bf4" \
-  LNDR_UCAC_RUB="0x14eb816e20af23ef81cc1deeba71d8642edcb621" \
   LNDR_UCAC_TRY="0xfe2bbfbe30f835096ccbc9c12a38ac749d8402b2" \
+  LNDR_UCAC_USD="0x7899b83071d9704af0b132859a04bb1698a3acaf" \
+  LNDR_UCAC_VND="0x815dcbb2008757a469d0daf8c310fae2fc41e96b" \
+  LNDR_UCAC_INR="0x24c7763f5a10370f5b431926f94daa53398182cc" \
```

Comment:
Looks like you started organizing these alphabetically, which is excellent, but then appended `LNDR_UCAC_INR` instead of putting it in alphabetical order as well.